bench/perf_run_py/perf_run.py

```diff
@@ -320,13 +321,17 @@ def run_perf_tests(config: PerfConfig,
             if args.verbose:
                 print("Batch test {}: Arguments: {}, Operations: {}".
                       format(index,  content["arguments"], content["operations"]))
-                perf_stats = PerfStatCollection(content["operations"])
-                if not args.reuse:
-                    run_test_wrapper(config=config, index=index, arguments=content["arguments"])
-                reported_stats += process_results(config, perf_stats, index=index)
+            perf_stats = PerfStatCollection(content["operations"])
```

Comment:
I've also fixed indentation here as this section was incorrectly hidden behind the `if args.verbose:` branch.
The new code is 
```
if (config.improved_accuracy):
    arguments = configure_for_extra_accuracy(config, arguments)
```
similar to below when there's no batch file